stdlib/src/builtin/object.mojo

```diff
@@ -591,9 +591,7 @@ struct _ObjectImpl(
             )
             return
         if self.is_func():
-            writer.write(
-                "Function at address " + hex(int(self.get_as_func().value))
-            )
```

Comment:
**Question** Is there a way we might be able to keep the `hex(..)` formatting here?

Reply:
**Followup** Never mind, I see you've changed the formatting of `UnsafePointer` to use `hex` by default as well.

That seems reasonable to me, though it would be great to call that out in the description as well :)